python/rmm/tests/test_rmm.py

```diff
@@ -914,3 +915,39 @@ def test_rmm_device_buffer_copy(cuda_ary, make_copy):
     result = db_copy.copy_to_host()
 
     np.testing.assert_equal(expected, result)
+
+
+@pytest.fixture
+def torch_allocator():
+    try:
+        from torch.cuda.memory import change_current_allocator
+    except ImportError:
+        pytest.skip("pytorch pluggable allocator not available")
+
+    try:
+        change_current_allocator(rmm.rmm_torch_allocator)
+    except RuntimeError:
+        pass
```

Comment:
What's the except/pass here for? Can we call `change_current_allocator` in an `else:` block from the first `try:`?

```python
try:
    from torch.cuda.memory import change_current_allocator
except ImportError:
    pytest.skip("pytorch pluggable allocator not available")
else:
    change_current_allocator(rmm.rmm_torch_allocator)
```

Alternatively can we use `pytest.importorskip`?

Reply:
Ah I think it's because a `RuntimeError` is raised if you set the torch allocator twice in the same session. Maybe we should use a session scoped fixture instead?